mautic/mautic
# Report group by and calculated fields

…nd aggregated functions to reports

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR adds custom contact fields to all data sources that have contact fields, it adds group by options to reports, it adds aggregated function to reports
#### Steps to test this PR:
Apply PR, test that reports work, and that you can add group by and aggregated fields to  your reports, also see all contact column in you data sources that use contact fields
1. 
2. 

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.
